editor.planx.uk/src/@planx/components/Pay/Public/FeeBreakdown/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface FeeBreakdown {
+  applicationFee: number;
+  total: number;
+  reduction: number;
+  vat: number | undefined;
+}
+
+export interface PassportFeeFields {
+  "application.fee.calculated": number;
+  "application.fee.payable": number;
+  "application.fee.payable.vat": number;
```

Comment:
Question - are you expecting `application.fee.payable.vat` to be an _input_ (eg already set in passport _before_ Pay is reached; this interface & unit tests indicate yes?) OR is it value that is _calculated by_ Pay and written to passport via Pay's `handleSubmit`? 

I was actually thinking the latter based on `VAT_RATE` defined earlier in the component, and a user journey like: 
- As an editor I have an "Apply VAT" toggle in my Pay component that is off my default. If I toggle it on then VAT is both included in displayed fee breakdown _and_ `application.fee.payable.VAT` amount will be written to passport on Pay `handleSubmit` (or sooner so capturable by Invite to pay journey) ? But I never need to create a separate SetValue or Calculate or similar myself which sets `application.fee.payable.vat` ?

Reply:
Initially (and in this PR) I was following [the pattern set here](https://editor.planx.dev/doncaster/pre-application-advice/nodes/_root/nodes/EIlRrlYGo6/edit) where Editors set this value before hitting the pay component though.

This is far from ideal though and on Monday we decided to - 
 - Do the calculation ourselves
 - Allow Editors to set a passport field of `application.fee.payable.includesVAT` which controls this

This is implemented here as a follow up PR - https://github.com/theopensystemslab/planx-new/pull/4042